draft/2023-07-19-this-week-in-rust.md

```diff
@@ -34,6 +34,7 @@ and just ask the editors to select the category.
 ### Newsletters
 
 ### Project/Tooling Updates
+* [We Just Released our Rust Teleconferencing System For Free (Free as in Freedom)](https://www.youtube.com/watch?v=KXfiWbCHlBw)
```

Comment:
Editing for length, adding link to the Rust codebase for readers' context/use
```suggestion
* [We Just Released our Rust Teleconferencing System for Free](https://www.youtube.com/watch?v=KXfiWbCHlBw) | [Repo](https://github.com/security-union/videocall-rs)
```

Reply:
Yes! this is perfect